tsconfig.json

```diff
@@ -6,7 +6,6 @@
     "emitDeclarationOnly": true,
     "exactOptionalPropertyTypes": true,
     "forceConsistentCasingInFileNames": true,
-    "importsNotUsedAsValues": "error",
```

Comment:
This will be removed from TypeScript soon, so it has to be replaced with rule `verbatimModuleSyntax`